meeting/meeting_june_8_2022.md

```diff
@@ -0,0 +1,56 @@
+# June 8th (6 PM CST)/9th (9 AM JST) Meeting 2022
+
+[A Telegram channel for people who ask you about where to join a channel](https://t.me/BitcoinDLCs)
+
+## Individual Updates (Sync)
+
+* **#Query(Nadav)**
+* **#Query(Chris)**
+* **#Query(Matt)**
+* **#Query(Tibo)**
+* **#Query(Lloyd)**
+* **#Query(Ben)**
+* **#Query(all)**
+
+## Implementation Updates (Sync)
+
+* bitcoin-s
+
+* rust-dlc
+
+* atomic finance
+
+* gun.fun
+
+* others?
+
+## Specification Writing
+
+**#Status_Update_Interrupt**
+
+* Message Segmentation
+  * https://github.com/discreetlogcontracts/dlcspecs/pull/192/
+  * Still needs review
+* Validation of offers and announcements
+  * https://github.com/discreetlogcontracts/dlcspecs/pull/193
+  * Still needs implementations
+* Multi-oracle with varying num_digits
+  * https://github.com/discreetlogcontracts/dlcspecs/pull/186
+  * Needs Review
+* Messaging and Serialization Breaking Changes
+  * https://github.com/discreetlogcontracts/dlcspecs/pull/163
+  * Updates
+    * **#Discussion**
```

Comment:
https://github.com/discreetlogcontracts/dlcspecs/pull/163#issuecomment-1150514743

Reply:
Also maybe worth discussing removing `cet_locktime` in this version of the protocol?